templates/product/modal/add_product.html

```diff
@@ -0,0 +1,22 @@
+{% extends "dashboard/base_modal.html" %}
+
+{% load i18n %}
+{% load materializecss %}
+
+{% block title %}
+{% trans "Product type" context "Modal select product classes title" %}
+{% endblock %}
+
+{% block content %}
+{{ form|materializecss }}
+{% endblock %}
+
+{% block action_url %}
+{% url "dashboard:product-select-classes" %}
+{% endblock %}
+
+{% block form_class %}form-async{% endblock %}
+
+{% block primary_action %}
+{% trans "Create new" context "Dashboard add action" %}
```

Comment:
This label was here before, but could we change it to "Create"? This will be more consistent with other buttons in the UI.

Reply:
Will do.